docs/v1/api.md

```diff
@@ -1,43 +1,63 @@
 # V1 API
 
-This document contains the Kubernetes resource types for the v1 service catalog.
+This document contains resource types and their standard usage in v1 service
+catalog. Although this API will be implemented in Kubernetes, other systems
+are not precluded from implementing it as well.
 
 # Resource Types
 
 This section lists descriptions of the resources in the service catalog API.
 
-_ __Note:__ All names herein are tentative, and should be considered placeholders
+*__Note:__ All names herein are tentative, and should be considered placeholders
 and used as placeholders for purposes of discussion only. This note will be
-removed when names are finalized._
+removed when names are finalized.*
+
+## `Broker`
+
+This resource is created by an administrator to instruct the service catalog's
+controller event loop to do the following:
+
+1. Make a request against a given CF service broker's catalog endpoint
+   (`GET /v2/catalog`)
+2. Translate the response to a list of `ServiceClass`es
+3. Write each translated `ServiceClass` to stable storage
```

Comment:
Does `stable storage` have some exact definition, I am more familiar with the phrase `persistent storage`, but I could not say it means anything more than "survives a restart of the program".